coa-helm/templates/cromwell.yaml

```
@@ -36,16 +38,18 @@ status: {}
 {{ end }}
 
 ---
-
-{{ include "terra-batch-libchart.cromwell-service" (list . "coa-helm.cromwell-service") }}
+{{- if .Values.cromwell.coaEnabled }}
+{{ include "terra-batch-libchart.cromwell-service" (list . "cromwell-helm.cromwell-service") }}
```

I think this still isn't right, the line prior to this PR was:
```suggestion
{{ include "terra-batch-libchart.cromwell-service" (list . "coa-helm.cromwell-service") }}
```

---

updated!